storage/http_client.go

```diff
@@ -367,8 +380,29 @@ func (c *httpStorageClient) DeleteBucketACL(ctx context.Context, bucket string,
 	return errMethodNotSupported
 }
 func (c *httpStorageClient) ListBucketACLs(ctx context.Context, bucket string, opts ...storageOption) ([]ACLRule, error) {
-	return nil, errMethodNotSupported
+	s := callSettings(c.settings, opts...)
+	var acls *raw.BucketAccessControls
+	var err error
+	err = run(ctx, func() error {
+		req := c.raw.BucketAccessControls.List(bucket)
+		configureACLCall(ctx, s.userProject, req)
+		acls, err = req.Do()
+		return err
+	}, s.retry, true)
+	if err != nil {
+		return nil, err
+	}
+	return toBucketACLRules(acls.Items), nil
 }
+func configureACLCall(ctx context.Context, userProject string, call interface{ Header() http.Header }) {
```

Comment:
Add a newline between methods (did you run gofmt?). Also, let's add a short comment noting what this method does to clarify the reflection magic (which was taken from the existing implementation for HTTP, btw)

Reply:
Done